src/commcare_cloud/ansible/roles/formplayer/templates/logback-spring.xml.j2

```diff
@@ -7,8 +7,6 @@
         </encoder>
     </appender>
 
-    <appender name="SENTRY" class="io.sentry.logback.SentryAppender" />
```

Comment:
this is causing duplicate sentry events

Reply:
Just curious what prompted adding this [here](https://github.com/dimagi/commcare-cloud/pull/4549)?